index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="styles/style.css" rel="stylesheet">
+    <link href="styles/synesthesia.css" rel="stylesheet">
+</head>
+
+
+<body>
+
+    <header>
+
+        <h1 id = "home-text" class = "page-text"> Hi, I'm Nikki Torab </h1>
+
+        
+    </header>
+
+    
+
+    <section class = "elephant">
+        <img src="images/elephant.gif" alt="Elephant-Walking-Gif">
+        <hr id = "home-line"/>
+    </section>
+
+
+
+    <nav class = "nav-bar top-nav">
+        <a class = "h" href = "index.html"> Home </a>
+        <a class = "s" href = "pages/synesthesia.html"> Synesthesia </a>
+        <a class = "a" href = "pages/about.html"> About </a>
+        <a class = "p" href = "pages/portfolio.html"> Projects </a>
+        <a class = "a" href = "pages/art.html"> Art </a>
```

Comment:
It's cool that you styled these with the color associated with their first letter.  😃